tests/attention/test_cudnn_prefill.py

```
@@ -214,6 +216,18 @@ def test_cudnn_prefill_fp8(
     torch.manual_seed(seed)
     device = "cuda:0"
 
+    major, minor = get_compute_capability(device)
```

Ruff (0.14.10)

Unpacked variable `minor` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)